pyproject.toml

```diff
@@ -73,6 +73,7 @@ fastapi-utils = { version = "~0.2.1", optional = true }
 orjson = { version = "~3.8.3", optional = true }
 Jinja2 = { version = "*", optional = true }
 ipinfo = { version = ">=4.4.3", optional = true }
+slowapi = { version = ">=0.1.9", optional = true }
```

Comment:
Don't we have a clash between our pytest and the one specified in slowapi? 

`pytest = "~=6.2.5"`

https://github.com/laurentS/slowapi/blob/6547679463b6a6fa87d9e254a8558bbc6075e263/pyproject.toml#L31C1-L31C19

Reply:
@strickvl this might not be the case anymore if we switch to fastapi_limiter which Andrei
 told me he would look into

Reply:
fastapi_limiter uses redis to perform rate limiting. Redis needs to be installed separately. This is not something that we want to make mandatory.